cclib/parser/utils.py

```diff
@@ -69,8 +69,11 @@ def convertor(value, fromunits, tounits):
 
     _convertor = {
 
-        "Angstrom_to_bohr":   lambda x: x * 1.8897261245,
-        "bohr_to_Angstrom":   lambda x: x * 0.5291772109,
+        "au_to_fs": lambda x: x * 0.02418884,
```

Comment:
Is there some better name for this here than "au"?

Reply:
There's no name for the unit. `timeau` or `autime`?

Reply:
Either is fine, but maybe with an additional underscore?